tests/test/scheduler/test_executor.cpp

```diff
@@ -29,238 +29,224 @@ namespace tests {
 std::atomic<int> restoreCount = 0;
 std::atomic<int> resetCount = 0;
 
-class TestExecutor final : public Executor
+TestExecutor::TestExecutor(faabric::Message& msg)
+  : Executor(msg)
+{}
+
+TestExecutor::~TestExecutor() {}
```

Comment:
This should also give a clang-tidy warning about using `= default` instead I think.